app/models/author.rb

```diff
@@ -1,6 +1,6 @@
 class Author < ActiveRecord::Base
   has_paper_trail on: [:destroy]
-  validates :cap_profile_id, uniqueness: true
+  validates :cap_profile_id, uniqueness: true, presence: true
```

Comment:
- curious about a corresponding db-migration for `NOT NULL` on that field.

Reply:
I don't care about that right now.  It doesn't have the same value as DB uniqueness constraint, because presence can be validated by the model without touching the DB.  (There is also no race condition for presence.)

Reply:
:ok_hand: